Kimjin/3주차 과제/과제1/style.css

```
@@ -0,0 +1,83 @@
+body {
+  background-color: rgb(139, 186, 204);
+}
+
+div {
+  margin: 50px auto;
+  width: 500px;
+  display: flex;
+  justify-content: center;
+  border: 3px solid white;
+  border-radius: 10px;
+  background-color: rgb(216, 243, 255);
+}
+
+.box2 {
+  height: 150px;
+  margin-top: -30px;
+}
+
+.one1 {
+  height: 45px;
+  width: 45px;
+  border-radius: 50%;
+  border: none;
+  background-color: rgb(65, 65, 150);
+  transform: all 1s ease-in;
```

변화하는 과정에 대한 효과를 주고 싶은 거였죠? 그럴 때는 `transform`이 아니라 `transition`을 사용합니다!
```suggestion
  transition: all 1s ease-in;
```